.hintrc

```diff
@@ -0,0 +1,18 @@
+{
+    "connector": {
+      "name": "local",
+      "options": {
+        "pattern": ["**", "!.git/**", "!node_modules/**"]
+      }
+    },
+    "extends": ["development"],
+    "formatters": ["stylish"],
+    "hints": [
+      "button-type",
+      "disown-opener",
+      "html-checker",
+      "meta-charset-utf-8",
+      "meta-viewport",
+      "no-inline-styles:error"
+    ]
+  }
```

Comment:
- [ ] Nice to have linters display green as needed however, it seems like you didn't follow the Microverse [template](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.eslintrc.json) to setup the linters, please go back and set up again using the template provided.

Reply:
Yes, I copy and paste the linters of microverse!

Reply:
This issue is not well resolved 😿

Reply:
So what do I have to do? I copied the template to setup the linters

Reply:
![image](https://user-images.githubusercontent.com/74747182/128397871-13a0720f-55a1-48b9-8763-eb51d431fbe6.png)

Reply:
Hi, here's is the link to the current `.hintrc`  https://github.com/microverseinc/linters-config/blob/master/html-css/.hintrc

Reply:
Kindly check carefully and you'll notice that it's different from what you have